js/index.js

```diff
@@ -228,25 +228,29 @@ function submitForm() {
         dataType: 'json',
     }).then(function(res) {
         console.log(res);
+        var count = res.queries_count;
+        var res = 5 - count;
+        showAlert(`上傳成功！ 您已經上傳${count}次，還有${res}可以上傳。`);
```

Comment:
Don't use new es6 syntax here, IE (<= 8) is a big problem, actually, IE 11 doesn't support it yet.

See https://kangax.github.io/compat-table/es6/ > template literals